index.html

```diff
@@ -33,12 +35,171 @@ <h1 class="header">
         </p>
         <a href="" class="button">LET'S CONNECT</a>
         <ul class="profiles">
-          <li><a href="https://twitter.com/MAleemHashmi1"><img src="twitter.png"></a></li>
-          <li><a href="https://www.linkedin.com/in/muhammad-aleem-hashmi-09658a171/"><img src="linkedin.png"></a></li>
-          <li><a href="https://github.com/MAleemH"><img src="github.png"></a></li>
+          <li><a href="https://twitter.com/MAleemHashmi1"><img src="images/twitter.png"></a></li>
+          <li><a href="https://www.linkedin.com/in/muhammad-aleem-hashmi-09658a171/"><img src="images/linkedin.png"></a></li>
+          <li><a href="https://github.com/MAleemH"><img src="images/github.png"></a></li>
         </ul>
-      </section>
+      </div>
+    </section>
+    <!-- "Works" section -->
+    <section class="works-section">
+
+      <div class="grid-container">
+
+        <!-- item 1: tonic -->
+        <div class="grid-item">
+          
+          <div class="portfolio-image-1"></div>
+          <h2 class="project-title">Tonic</h2>
+
+          <div class="canopy">
+            <h6>CANOPY</h6>
+            <ul class="canopy-ul">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <a href="" class="button btn-border">See Project</a>
+
+        </div>
+
+        <!-- item 2: multi-post stories -->
+        <div class="grid-item">
+
+          <div class="portfolio-image-2"></div>
+          <h2 class="project-title">Multi-Post <br> Stories</h2>
+
+          <div class="canopy">
+            <h6>CANOPY</h6>
+            <ul class="canopy-ul">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <a href="" class="button btn-border">See Project</a>
+
+        </div>
+        <!-- item 3: tonic -->
+        <div class="grid-item">
+
+          <div class="portfolio-image-3"></div>
+          <h2 class="project-title">Tonic</h2>
+
+          <div class="canopy">
+            <h6>CANOPY</h6>
+            <ul class="canopy-ul">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <a href="" class="button btn-border">See Project</a>
+
+        </div>
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible like color, font, size, etc.
- [ ] Please increase the font size of the `h2` tag So it looks like the design.
- [ ] Please use the same border for the buttons as it is different from the Figma design
- [ ] Please fix the spacing by looking at the Figma design.
- [ ] I would suggest you please do this for every card.
### Your design
 
![your](https://user-images.githubusercontent.com/81727988/205269454-732022d9-76d0-4244-874f-5a8990da9545.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/205269519-2f5e81cc-15aa-4952-be97-51a19fc4183f.png)